pages/_app.js

```diff
@@ -20,18 +22,34 @@ const fetcher = async (url) => {
   return res.json();
 };
 
+const URL_ = "/api/episodes";
+export const episodes = atom([]);
+
 export default function App({ Component, pageProps }) {
+  const { data, isLoading, error } = useSWR(URL_, fetcher);
+  const [, setAllEpisodes] = useAtom(episodes);
+
+  if (data) setAllEpisodes(data);
```

Comment:
setting states like this is best done in a useEffect with the `data` inside the dependency array.

Reply:
changed it

Reply:
In the preview, it takes a lot longer now to render the homepage. Maybe I did something wrong with useEffect

Reply:
fixed it :D